source/commands.txt

```diff
@@ -76,10 +76,8 @@ To learn more about connecting to a deployment, see :ref:`vsce-connect`.
      - Select and remove a connection from your list of connections
        in |vsce|.
 
-       .. note::
-
-          Removing your current active connection also closes that
-          connection.
+       Removing your current active connection also closes that
+       connection.
 
```

Comment:
I can't comment on the applicable lines, but on this file and on `settings.txt`, build errors are being caused by the `|vsce-short|` substitution. Can you check to make sure this variable exists in `snooty.toml`? If not, replace the variable with `|vsce|` !

Reply:
@ajhuh-mdb Tried to fix the build errors seems like the new command is 'vscode-short'. Hopefully that works!